src/test/test.rs

```diff
@@ -107,7 +106,13 @@ mod tests {
                     for server in output.unwrap().iter() {
                         output_dnsservers.push(format!("{}", server));
                     }
-                    assert_eq!(expected_dnsservers, output_dnsservers);
+                    if rustc_version::version().unwrap()
+                        >= rustc_version::Version::parse("1.72.0").unwrap()
+                    {
+                        assert_eq!(vec!["127.0.0.1", "::2"], output_dnsservers);
+                    } else {
+                        assert_eq!(vec!["127.0.0.1", "::0.0.0.2"], output_dnsservers);
+                    }
```

Comment:
Wouldn't it be much better to just directly compare based on the actual IpAddr struct instead of the string representation, then we do not need to worry about this I think.

Reply:
Sounds like a good idea, let me check that.